docs/user/manual.adoc

```diff
@@ -288,11 +285,37 @@ interface Crate {
     edition: "2015" | "2018";
     /// Dependencies
     deps: Dep[];
+    /// Should this crate be treated as a member of current "workspace".
+    ///
+    /// By default, inferred from the `root_module` (members are the crates which reside
+    /// inside the directory opened in the editor).
+    ///
+    /// Set this too `false` for things like standard library and 3rd party crates to
+    /// enable performance optimizations (rust-analyzer assumes that non-member crates
+    /// don't change).
+    is_workspace_member?: boolean;
```

Comment:
This is probably the most interesting key, previously we assumed that all crates are "local", which made things slower for incremental changes.

Reply:
Do we need to do anything if we _do_ end up changing crates that are marked with false?